plugins/binaryninja/tests/test_native_plugin.py

```diff
@@ -142,5 +143,201 @@ def test_global(self) -> None:
+        4198428,
+        4198433,
+        4198440,
+        4198447,
+        4198480,
+        4198528,
+        4198568,
+        4198576,
+        4198632,
+        4198640,
+        4198653,
+        4198679,
+        4198684,
+        4198704,
+        4198713,
+        4198733,
+        4198784,
+        4198836,
+        4198848,
+        4198862,
+        4198896,
+        4198902,
+        4198910,
+        4198923,
+        4198963,
+        4198979,
+        4199006,
+        4199016,
+        4199023,
+        4199034,
+        4199042,
+        4199055,
+        4199181,
+        4199192,
+        4199214,
+        4199229,
+        4199328,
+        4199343,
+        4199362,
+        4199370,
+        4199379,
+        4199392,
+        4199428,
+        4199438,
+        4199456,
+        4199490,
+        4199520,
+        4199536,
+        4199565,
+        4199576,
+        4199581,
+        4199600,
+        4199621,
+        4199641,
+        4199645,
+        4199687,
+        4199703,
+        4199712,
+        4199729,
+        4199808,
+        4199824,
+        4199828,
+        4199830,
+        4199861,
+        4199936,
+        4200043,
+        4200096,
+        4200123,
+        4200160,
+        4200172,
+        4200181,
+        4200192,
+        4200225,
+        4200236,
+        4200307,
+        4200350,
+        4200442,
+        4200447,
+        4200455,
+        4200460,
+        4200484,
+        4200496,
+        4200544,
+        4200576,
+        4200912,
+        4200954,
+        4200963,
+        4200976,
+        4200992,
+        4200998,
+        4201019,
+        4201168,
+        4201217,
+        4201236,
+        4201241,
+        4201255,
+        4201271,
+        4201312,
+        4201339,
+        4201424,
+        4201443,
+        4201465,
+        4201488,
+        4201524,
+        4201537,
+        4201552,
+        4201563,
+        4201584,
+        4201595,
+        4201598,
+        4201603,
+        4201696,
+        4201704,
+        4201708,
+        4201744,
+        4201803,
+        4201814,
+        4201833,
+        4201904,
+        4201917,
+        4201944,
+        4201982,
+        4202007,
+        4202016,
+        4202020,
+        4202025,
+        4202029,
+        4202038,
+        4202058,
+        4202073,
+        4202078,
+        4202079,
+        4202094,
+        4202128,
+        4202138,
+        4202161,
+        4202224,
+        4202237,
+        4202288,
+        4202293,
+        4202307,
+        4202327,
+        4202384,
+        4202392,
+        4202400,
+        4202410,
+        4202441,
+        4202453,
+        4202465,
+        4202480,
+        4202633,
+        4202648,
+        4202656,
+        4202661,
+        4202705,
+        4202784,
+        4202804,
+        4202832,
+        4202912,
+        4203138,
+    }
+    # Init libc seems to behave differently between manticore and native, ignore blocks from it
```

Comment:
This might be due to different cpuid results maybe?

Reply:
My understanding is that it's something to do with environment variables, but I can't pinpoint the exact issue

Reply:
Would you mind writing that in the comment here please, unless you're really unsure? Might help someone later to debug.